### PR TITLE
Add CSP health check article

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Add CSP health check article

Fixes: https://github.com/umbraco/UmbracoDocs/issues/6336

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
Umbraco > 14.3 

## Deadline (if relevant)
Thursday, October 3 2024 (release Umbraco 14.3)

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @erikjanwestendorp ! 💪 

I'll make sure we get the reviewed and ready for the 14.3 release!

File: 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Health check: Content Security Policy (CSP)
```
Comment (sofietoft) [inline 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md:1]: No need for adding "Health Check" here - let's keep it consistent with the other Health Check guides 💪 

```suggestion
# Content Security Policy (CSP)
```

File: 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        .Self())
+    .ConnectSources(s => s
+        .Self())
+    .FrameSources(s => s
+        .Self()));
+```
+
+### Adding a Content Security Policy (CSP) using manual middleware
+
+Avoid third-party library dependency by using custom middleware added to the request pipeline.
```
Comment (sofietoft) [inline 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md:42]: Suggested a small rewrite here, to make it a little more clear what the code snippet below does.
Let me know if it does not make sense!

```suggestion
Avoid third-party library dependencies by using custom middleware added to the request pipeline as shown below.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
This is great @erikjanwestendorp ! 💪 

Could you add the article to the SUMMARY.md file for two versions as well? 🤞 Otherwise it will not show in the published docs.
Also, does it make sense to link to this new article from the [Cross-site scripting Protection article](https://docs.umbraco.com/umbraco-cms/~/revisions/W8LxCySfNCUwz7TdJPNi/extending/health-check/guides/crosssitescriptingprotection), which is currently somewhat empty? 🤔

File: 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md]: I updated the PR with your suggestion but 'Health check: ' is used in the other articles right?

For example: 

image

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for the review! 😄 Just updated the PR with your suggestions.

**Review by sofietoft — APPROVED** [review]
Looks great @erikjanwestendorp !
And thanks for adding it to the 15 docs as well - much appreciated!

I'll get it merged.

**Pull request merged**